pkg/advancedapi/advancedApi_test.go

```diff
@@ -313,48 +313,6 @@ func Test_can_add_control_delegation_proof(t *testing.T) {
 	assert.Check(t, doc.DelegateControl[0].Revoked == false)
 }
 
-func Test_can_remove_control_delegation(t *testing.T) {
```

Comment:
Should all these `can_remove...` tests be left in and inverted to `cannot_remove...`?

Reply:
These tests only call the `Remove...` functions that have been removed. The tests for checking keys cannot be remove, has been moved to pkg/register/document_test.go